dashboard/app/models/hidden_stage.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: hidden_stages
+#
+#  id         :integer          not null, primary key
+#  section_id :integer
+#  stage_id   :integer
+#
+# Indexes
+#
+#  index_hidden_stages_on_section_id  (section_id)
+#  index_hidden_stages_on_stage_id    (stage_id)
+#
+
+require 'cdo/section_helpers'
```

Comment:
I probably copy/pasted this file and accidentally left his in here? I can try without